.gitignore

```diff
@@ -1,3 +1,4 @@
 target
 .sbtopts
 project/.sbt
+.idea/
```

Comment:
Revert this please

Reply:
Hmm, I thought that should be fine. It might have to be ignored one day anyway. **I reverted the change**

Reply:
I added a note in master with the reason it shouldn't be added